src/app/home/home.component.ts

```diff
@@ -65,8 +65,9 @@ export class HomeComponent implements OnInit, AfterViewInit {
   }
 
   backgroundRoute() {
-    const routesWithBackground = [ 'resources' ];
-    const routesWithoutBackground = [ 'resources/add', 'resources/view' ];
+    const routesWithBackground = [ 'resources', 'users' ];
+    // Leaving the exception variable in so we can easily use this while still testing backgrounds
+    const routesWithoutBackground = [];

```

Comment:
You should not remove `resources/add` and `resources/view` from this array.  These routes still need to be included as exceptions with a white background until their views are updated.

Reply:
Also `users/profile` and `users/update` should be added to the `routesWithoutBackground`